<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>789</width>
    <height>529</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>789</width>
    <height>529</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">@import url('https://fonts.googleapis.com/css2?family=Roboto:wght@300&amp;display=swap');</string>
  </property>
  <widget class="QWidget" name="mainContainer">
   <property name="styleSheet">
    <string notr="true">#mainContainer{
background-color:#272741;
}
#imageContainer{
background-color:#4A4A6C;
}</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QWidget" name="imageContainer" native="true">
      <property name="styleSheet">
       <string notr="true">#toolsContainer{
background-color:#CFCFCF;
}</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QWidget" name="widget_2" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QWidget" name="toolsContainer" native="true">
            <property name="minimumSize">
             <size>
              <width>87</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>87</width>
              <height>16777215</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="image" native="true">
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <item>
              <widget class="QLabel" name="label">
               <property name="maximumSize">
                <size>
                 <width>621</width>
                 <height>345</height>
                </size>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="pixmap">
                <pixmap>images/road.jpg</pixmap>
               </property>
               <property name="scaledContents">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="bottomArea" native="true">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>140</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">
#labelInstruction{
color:white;
}</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QLabel" name="labelInstruction">
         <property name="text">
          <string>  INSTRUCTION: Paint the road with red paint using the tools. </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="roadName" native="true">
         <property name="styleSheet">
          <string notr="true">#labelRoad{
color:#8F8F9C;
font-size:12px;

}</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="labelRoad">
            <property name="text">
             <string>Road Name</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="roadTextBox">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>29</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>636</width>
              <height>16777215</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item alignment="Qt::AlignHCenter">
        <widget class="QWidget" name="buttonsContainer" native="true">
         <property name="styleSheet">
          <string notr="true">QPushButton{
background: #1D1F32;
border: 1px solid #678ADD;
border-radius: 15px;
color:white;
}</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <property name="spacing">
           <number>90</number>
          </property>
          <item>
           <widget class="QPushButton" name="btnCancel">
            <property name="minimumSize">
             <size>
              <width>125</width>
              <height>36</height>
             </size>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="text">
             <string>Cancel</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnDone">
            <property name="minimumSize">
             <size>
              <width>125</width>
              <height>36</height>
             </size>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="text">
             <string>Done</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
